app/models/subnet/ipv4.rb

```diff
@@ -28,6 +28,12 @@ def self.supported_ipam_modes
     [:dhcp, :db, :none]
   end
 
+  def known_ips
```

Comment:
Seems like you can get rid of this and the `ipv6` equivalent by creating a class method `ip_sym` or something similar in each class and just use that in the parent class